Standalone/Standalone/NativeTableViewController.m

```diff
@@ -0,0 +1,138 @@
+//
+//  NativeSampleTableViewController.m
+//  Standalone
+//
+//  Copyright © 2020 Criteo. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#import "NativeTableViewController.h"
+#import "AdConfigurations.h"
+
+static const NSUInteger nativeAdIndex = 3;
+static const NSUInteger cellHeight = 140;
+
+@interface NativeTableViewController () <CRNativeLoaderDelegate>
+
+@property (strong, nonatomic) NSMutableArray *tableViewItems;
+@property (strong, nonatomic) NativeAdTableViewCell *nativeAdCell;
+@property (strong, nonatomic) CRNativeLoader *nativeLoader;
+
+@end
+
+@implementation NativeTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.tableViewItems = [[NSMutableArray alloc] init];
+    
+    [self.tableView registerNib:[UINib nibWithNibName:@"ContentItemTableViewCell" bundle:nil] forCellReuseIdentifier:@"contentCell"];
+    [self.tableView registerNib:[UINib nibWithNibName:@"NativeAdTableViewCell" bundle:nil] forCellReuseIdentifier:@"nativeAdCell"];
+    
+    [self addContent];
+    [self addNativeAd];
+}
+
+#pragma mark - Table view data source
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
+    return 1;
+}
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+    return self.tableViewItems.count;
+}
+
+- (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPath *)indexPath {
+    return cellHeight;
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+    if ([self.tableViewItems[indexPath.row] isKindOfClass:[NativeAdTableViewCell class]]) {
+        UITableViewCell *cell = [self.tableView dequeueReusableCellWithIdentifier:@"nativeAdCell" forIndexPath:indexPath];
+        for (UIView *subview in cell.contentView.subviews) {
```

Comment:
Why removing subviews / adding the native ad view there?
That should not be necessary and most likely is counterperformant.

The only thing that should be done is to set `nativeAdCell.ad` and other props?

Reply:
Removed the for loop.